File: java/src/main/java/com/smeltery/service/config/properties/WebSecurity.java
package com.smeltery.service.config.properties;

import java.util.Arrays;
import java.util.List;

public class WebSecurity {
	private String corsAllowedOrigins;

	public List<String> getCorsAllowedOrigins() {
		return Arrays.asList(corsAllowedOrigins.split(","));
	}
}

File: kotlin/src/main/kotlin/com/ibm/service/config/properties/WebSecurity.kt
package com.smeltery.service.config.properties

class WebSecurity {
	private val corsAllowedOrigins: String? = null

	fun getCorsAllowedOrigins(): List<String> {
		return listOf(
			*corsAllowedOrigins!!
				.split(",".toRegex()).dropLastWhile { it.isEmpty() }
				.toTypedArray()
		)
	}
}
